Source/Widgets/Viewer/Viewer.js

```diff
@@ -176,6 +180,55 @@ define([
         });
     }
 
+    function enableVRUI(viewer, enabled) {
+        var geocoder = viewer._geocoder;
+        var homeButton = viewer._homeButton;
+        var sceneModePicker = viewer._sceneModePicker;
+        var baseLayerPicker = viewer._baseLayerPicker;
+        var animation = viewer._animation;
+        var timeline = viewer._timeline;
+        var fullscreenButton = viewer._fullscreenButton;
+        var infoBox = viewer._infoBox;
+        var selectionIndicator = viewer._selectionIndicator;
+
+        var display = enabled ? 'none' : 'block';
```

Comment:
OK.  I would expect most toolbar buttons to be `inline-block`, not `block`, since they are placed horizontally.  I'll check whether just setting them all to `inline-block` across the board is OK.

The selectionIndicator would be great to have, but remember everything needs to be duplicated.  We'd need one for the left eye, and another one for the right.